source/uwp/AdaptiveCardsObjectModel/lib/AdaptiveNumberInput.cpp

```diff
@@ -9,9 +9,21 @@ namespace winrt::AdaptiveCards::ObjectModel::Uwp::implementation
 {
     AdaptiveNumberInput::AdaptiveNumberInput(const std::shared_ptr<::AdaptiveCards::NumberInput>& sharedNumberInput)
     {
-        Min = sharedNumberInput->GetMin();
-        Max = sharedNumberInput->GetMax();
-        Value = sharedNumberInput->GetValue();
+        if (sharedNumberInput->GetMin().has_value())
```

Comment:
Can these be extracted to a function? Like:

```c++
template<typename T> winrt::Windows::Foundation::IReference<T> box_or_default(std::optional<T> const& t)
{
    if (t.has_value())
    {
        return winrt::box_value(t.value()).as<Windows::Foundation::IReference<T>>();
    }
    else
    {
        return nullptr;
    }
}
```

Reply:
This goes away with the property_opt fix